src/signalfx-rest.js

```diff
@@ -6,8 +6,9 @@ const request = require('requestretry');
  */
 
 module.exports = class SignalFxRest {
```

Comment:
Note that there's already a `FakeSignalFxRest` in `test/helper.js`, so no need to add a mock option here..